src/root/.rtorrent.rc

```diff
@@ -37,4 +37,4 @@ log.add_output = "storage_info", "rtorrent"
 
 use_udp_trackers = yes
 
-#system.file_allocate.set = yes
\ No newline at end of file
+#system.file_allocate.set = yes
```

Comment:
This line was actually not changed, only seems like the newline sign was removed by my editor.